src/frontends/pytorch/src/op_table.cpp

```diff
@@ -789,6 +789,7 @@ const std::unordered_map<std::string, CreatorFunction> get_supported_ops_fx() {
         {"aten.argmin.default", op::translate_argmin},
         {"aten.as_strided.default", op::translate_as_strided},
         {"aten.as_strided_.default", op::translate_as_strided},
+        {"aten.as_strided_copy.default", op::translate_as_strided},
```

Comment:
can you please add layer tests for newly added translators?

Reply:
I added layer tests for all new ops except aten.slice_copy. Initially created aten.slice_copy is converted into multiple select ops before translation with torch.compile. We are planning to implement layer tests in executorch backend side as well which will cover all new translators.